lib/super_good/solidus_taxjar/backfill_transactions.rb

```diff
@@ -0,0 +1,8 @@
+module SuperGood
```

Comment:
I would generally make this class and actually implement it in the commit before this one. This commit is already quite long.